File: netlify/functions/health.py
"""
Health check endpoint for Netlify
"""
import json

def handler(event, context):
    """Health check handler"""
    return {
        'statusCode': 200,
        'headers': {'Content-Type': 'application/json'},
        'body': json.dumps({'status': 'ok', 'message': 'Pipedrive-Katana sync is running'})
    }

File: netlify/functions/health.js
exports.handler = async () => ({
  statusCode: 200,
  headers: {
    'Content-Type': 'application/json',
    'Cache-Control': 'no-store',
  },
  body: JSON.stringify({
    status: 'retired',
    retired: true,
    message: 'The legacy Pipedrive-to-Katana sync is retired.',
  }),
});
